### DISCUSSION TITLE
Custom control of grids from another program

### DISCUSSION THREAD

**Discussion by shub2023** [vzhd1701/gridplayer#167] [Q&A]
Many thanks and kudos to vzhd1701 and your team for building this wonderful player.

I wanted to check if it is possible to control the launch instance from an another program. 

My use case require Matlab to pass argument to Gridplayer about the grid layout (ie 2 row and 3 column in the player) and the video files for each grid. This might vary is each time, ie it could be 2x3 grid in one instance and in subsequent instance it could be 4x4 grid.

Thanks.

**Comment by vzhd1701** [comment]
I'll add CLI in future. For now the solution can be generating playlist file in Matlab, saving it in temp directory and passing it's path to the GridPlayer.

**Reply by shub2023** [reply]
Thank you.
